examples/kubernetes/clustermesh/global-service-example/cluster1.yaml

```diff
@@ -12,6 +12,21 @@ spec:
   selector:
     name: rebel-base
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: rebel-base-headless
+  annotations:
+    service.cilium.io/global: "true"
+    service.cilium.io/global-sync-endpoint-slices: "true"
```

Comment:
I added this in examples but didn't added the corresponding explanation with DNS queries in `
Deploying a Simple Example Service` since the `deployment/x-wing` image doesn't seems to have any tools to query dns (i.e.: dig),